tests/gtest/test_population_Unfished_Initial.cpp

```diff
@@ -79,42 +80,57 @@ namespace
                     // or to CalculateUnfishedNumbersAA(index_ya, a-1);?
```

Comment:
should we be removing all these commented out code chunks when we merge into 180-feat-population?

Reply:
I will move this comment over to the 180-feat-population PR